users/migrations/0002_user_profile_image.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.0.2 on 2022-02-17 10:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='user',
+            name='profile_image',
+            field=models.URLField(default=None, max_length=250),
```

Comment:
```suggestion
            field=models.URLField(default=None, max_length=250),
```

저희가 모델 필드 정의할때 default 값을 설정을 안해놔서 마이그레이션 시 임시로 None 값을 넣어놨습니다.

Reply:
음 default=None 이면 NULL로 들어갈텐데 이렇게되면 null=True가 되어야 하지 않을까요?

Reply:
희택님! 모델 필드에서 `default` 설정을 안해주셨다면, 모델을 수정하고 그 수정사항으로 migration 파일을 만들면 되지 않을까요?  지금 이렇게 수정하게되면, 나중에 `왜 모델엔 디폴트가 없는데 DB에는 디폴트가 설정되어있지?` 라는 의문이 생길 수 있습니다 ㅠ_ㅠ 내일의 나는 어제의 나를 기억 못하는 법..